.gitlab-ci.yml

```diff
@@ -40,6 +40,54 @@ test:lint:
   tags:
     - mender-qa-worker-generic-light
 
+test:types:
+  stage: test
+  image: python:3.11
+  needs: []
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+      when: always
+    - when: manual
+  before_script:
+    # Prepare SSH key
+    - eval $(ssh-agent -s)
+    - echo "$SSH_PRIVATE_KEY" | tr -d '\r' | ssh-add - > /dev/null
+    - mkdir -p ~/.ssh
+    - chmod 700 ~/.ssh
+    - ssh-keyscan github.com >> ~/.ssh/known_hosts
+    # Configure git
+    - git config --global user.email "mender@northern.tech"
+    - git config --global user.name "Mender Test Bot"
```

Comment:
I would rather document the changes to those values through git, since the trace custom env vars leave might be less obvious (also already in use [here](https://github.com/mendersoftware/integration/blob/master/.gitlab-ci.yml#L36-L37)